test/integration-spec.yaml

```
@@ -804,6 +804,30 @@
     httpsRequests:
       - "https://httpbin.org/basic-auth/foo/bar"
 
+# protocols
+- url: "https://127.0.0.1:8889"
+  label: "localhost but https version"
+  options:
+    ignore-ssl-errors: true
+  metrics:
+    mainDomainHttpProtocol: "http/1.1"
+    oldHttpProtocol: 1
+    mainDomainTlsProtocol: "TLS 1.1"
+    oldTlsProtocol: 1
+  offenders:
+    oldHttpProtocol:
+      - {
+          domain: "https://127.0.0.1:8889",
```

```suggestion
          domain: "https://127.0.0.1",
```

Should domain contain the port number?